lib/src/Pos/Util/UserSecret.hs

```diff
@@ -165,26 +201,26 @@ simpleUserSecret :: SecretKey -> FilePath -> UserSecret
 simpleUserSecret sk fp = def & usPrimKey .~ Just sk & usPath .~ fp
 
 instance Default UserSecret where
-    def = UserSecret [] Nothing Nothing Nothing "" Nothing
+    def = UserSecret [] Nothing Nothing [] "" Nothing
 
 -- | It's not network/system-related, so instance shouldn't be under
 -- @Pos.Binary.*@.
 instance Bi UserSecret where
   encode us = encodeListLen 4 <> encode (_usVss us) <>
                                       encode (_usPrimKey us) <>
                                       encode (_usKeys us) <>
-                                      encode (_usWallet us)
+                                      encode (_usWallets us)
```

Comment:
I understand that this PR isn't meant to properly™ redesign `UserSecret` including proper™ versioned serialization, but I should remind that you'll need to have it eventually (preferrably JSON serialization, as it's human-readable and thus debuggable). But this isn't my change request.